rfcs/20200609-logsigmoid/README.md

```diff
@@ -0,0 +1,58 @@
+# Introducing logsigmoid operation
+
+## Introduction
+
+Request comes from DGP graph lib community https://www.dgl.ai. Logsigmoid is
+used in AWS embedding models https://bit.ly/30kWI1f. Currently logsigmoid can be
+implemented with oneDNN by using two eltwise
+(https://oneapi-src.github.io/oneDNN/dev_guide_eltwise.html) operations
+logistic and log. Unfortunately current eltwise ops are not fusable.
```

Comment:
Can you please provide formulas for forward and backward (including backward computed via dst only)?

This may help decide whether we need a separate algorithm or not.

Reply:
I updated RFC with the formulas

Reply:
In case you are describing, this would demand combining eltwise linear with eltwise soft_relu (or opposite?) for backward, correct? How to combine those ops then? It doesn't seem to give the same output formula as combining logistic followed by log as @piotrchmiel proposed.